packages/seacas/libraries/ioss/src/Ioss_ZoneConnectivity.C

```diff
@@ -84,6 +85,179 @@ namespace Ioss {
+    }
+
+    if( this->m_ownerOffset != rhs.m_ownerOffset ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_ownerOffset MISMATCH ({}:{}:{} vs {}:{}:{})\n",
+                   IJK_list(this->m_ownerOffset), IJK_list(rhs.m_ownerOffset));
+      }
+      return false;
+    }
+
+    if( this->m_donorRangeBeg != rhs.m_donorRangeBeg ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_donorRangeBeg MISMATCH ({}:{}:{} vs {}:{}:{})\n",
+                   IJK_list(this->m_donorRangeBeg), IJK_list(rhs.m_donorRangeBeg));
+      }
+      return false;
+    }
+
+    if( this->m_donorRangeEnd != rhs.m_donorRangeEnd ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_donorRangeEnd MISMATCH ({}:{}:{} vs {}:{}:{})\n",
+                   IJK_list(this->m_donorRangeEnd), IJK_list(rhs.m_donorRangeEnd));
+      }
+      return false;
+    }
+
+    if( this->m_donorOffset != rhs.m_donorOffset ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_donorOffset MISMATCH ({}:{}:{} vs {}:{}:{})\n",
+                   IJK_list(this->m_donorOffset), IJK_list(rhs.m_donorOffset));
+      }
+      return false;
+    }
+
+    if( this->m_ownerGUID != rhs.m_ownerGUID ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_ownerGUID MISMATCH ({} vs {})\n",
+                   this->m_ownerGUID, rhs.m_ownerGUID);
+      }
+      return false;
+    }
+
+    if( this->m_donorGUID != rhs.m_donorGUID ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_donorGUID MISMATCH ({} vs {})\n",
+                   this->m_donorGUID, rhs.m_donorGUID);
+      }
+      return false;
+    }
+
+    if( this->m_ownerZone != rhs.m_ownerZone ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_ownerZone MISMATCH ({} vs {})\n",
+                   this->m_ownerZone, rhs.m_ownerZone);
+      }
+      return false;
+    }
+
+    if( this->m_donorZone != rhs.m_donorZone ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_donorZone MISMATCH ({} vs {})\n",
+                   this->m_donorZone, rhs.m_donorZone);
+      }
+      return false;
+    }
+
+    if( this->m_ownerProcessor != rhs.m_ownerProcessor ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_ownerProcessor MISMATCH ({} vs {})\n",
+                   this->m_ownerProcessor, rhs.m_ownerProcessor);
+      }
+      return false;
+    }
+
+    if( this->m_donorProcessor != rhs.m_donorProcessor ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_donorProcessor MISMATCH ({} vs {})\n",
+                   this->m_donorProcessor, rhs.m_donorProcessor);
+      }
+      return false;
+    }
+
+    if( this->m_sameRange != rhs.m_sameRange ) {
+      if( !quiet ) {
+        fmt::print(Ioss::DEBUG(), "ZoneConnectivity : m_sameRange MISMATCH ({} vs {})\n",
```

Comment:
The booleans should already print `true` or `false` without the ternary... I guess ternary is ok if you want them uppercased `True` or `False`, but adds  extra code...

Reply:
The ternary is left over from when this was erroneously a `printf`; allowing `fmt::print` to format the boolean absolutely makes sense.